src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -627,7 +627,7 @@ protected override void ProcessTypeReference (TypeReference type)
 
 			protected override void ProcessExportedType (ExportedType exportedType)
 			{
-				TypeDefinition td = exportedType.Resolve ();
+				TypeDefinition? td = exportedType.Resolve ();
```

Comment:
Why is it not calling TryResolve?

Reply:
Based off the information in https://github.com/dotnet/linker/issues/2260, I thought it would be best to use Cecil's Resolve here. In sweep step the metadata graph isn't stable, so we might resolve references to definitions that don't exist anymore. We don't have great testing around this, so I wanted to err on the side of caution and use the safer option for this. It doesn't seem to be a very hot path, so I think it would be okay for now.